src/authenticator.ts

```diff
@@ -6,6 +6,15 @@ import {AuthenticateOptions} from "./common";
 const TOKEN_LEEWAY = 30;
 const TOKEN_EXPIRY = 24*60*60;
 
+export interface TokenWithExpire {
+  token: string;
+  expire: number;
```

Comment:
Sounds ok. I usually go with `expiresAt`, but I don't mind the `Timestamp` suffix.